lib/constants.go

```diff
@@ -43,6 +43,10 @@ const (
 	MessagesToFetchPerInboxCall = 10000
 )
 
+// DisableNetworkManagerRoutines is used to disable the network manager routines.
+// This is useful for testing.
+var DisableNetworkManagerRoutines = false
```

Comment:
Please consider defining this flag as a field on the ConnectionController that you can set on each instance of the controller. Global flags typically lead to headaches down the line as the code gets more complicated